qiskit/circuit/quantumcircuit.py

```diff
@@ -1325,9 +1465,71 @@ def data(self, data_input: Iterable):
     def op_start_times(self) -> list[int]:
         """Return a list of operation start times.
 
-        This attribute is enabled once one of scheduling analysis passes
+        This attribute is enabled once one of the scheduling analysis passes
         runs on the quantum circuit.
 
+        Example:
+
+            .. plot::
+                :include-source:
+                :nofigs:
+
+                from qiskit import QuantumCircuit
+                from qiskit.providers.fake_provider import GenericBackendV2
+                from qiskit.transpiler import generate_preset_pass_manager
+
+                qc = QuantumCircuit(2)
+                qc.h(0)
+                qc.cx(0, 1)
+                qc.measure_all()
+
+                # Print the original circuit
+                print("Original circuit:")
+                print(qc)
+
+                # Transpile the circuit with a specific basis gates list and print the resulting circuit
+                backend = GenericBackendV2(2, basis_gates=['u1', 'u2', 'u3', 'cx'])
+                pm = generate_preset_pass_manager(optimization_level=1,
+                backend=backend, scheduling_method='alap')
```

Comment:
Minor formatting change with [black](https://github.com/psf/black)

```suggestion
                pm = generate_preset_pass_manager(
                    optimization_level=1, backend=backend, scheduling_method="alap"
                )
```